HPXMLtoOpenStudio/resources/hotwater_appliances.rb

```diff
@@ -242,7 +242,8 @@ def self.apply(model, runner, hpxml, weather, spaces, hot_water_distribution,
         wh_setpoint = Waterheater.get_default_hot_water_temperature(eri_version) if wh_setpoint.nil? # using detailed schedules
         avg_setpoint_temp += wh_setpoint * water_heating_system.fraction_dhw_load_served
       end
-      daily_wh_inlet_temperatures = calc_water_heater_daily_inlet_temperatures(weather, nbeds, hot_water_distribution, fixtures_all_low_flow)
+      daily_wh_inlet_temperatures = calc_water_heater_daily_inlet_temperatures(weather, nbeds, hot_water_distribution, fixtures_all_low_flow,
+                                                                               hpxml.header.sim_calendar_year)
```

Comment:
@shorowit This is needed because 2/29 actually gets forward translated now?

Reply:
Basically, yes. We were supplying 365 days and were getting schedule values thru 12/30 instead of 12/31 with the OS 3.6, so we got an EnergyPlus error that the schedule "has missing days in its schedule pointers". So now we supply 366 days if a leap year.

Reply:
Also, it's nice to see schedule values starting at 1/1 instead of 1/3 now! Thanks for fixing that.